snap/squashfs/squashfs.go

```diff
@@ -548,14 +554,20 @@ func (s *Snap) Build(sourceDir string, opts *BuildOpts) error {
 		cmd.Args = append(cmd.Args, "-all-root", "-no-xattrs")
 	}
 
-	return osutil.ChDir(sourceDir, func() error {
+	if err := osutil.ChDir(sourceDir, func() error {
 		output, err := cmd.CombinedOutput()
 		if err != nil {
 			return MksquashfsError{fmt.Sprintf("mksquashfs call failed: %s", osutil.OutputErr(output, err))}
 		}
 
 		return nil
-	})
+	}); err != nil {
+		return err
+	}
+
+	// grow the snap larger if it is smaller than the minimum snap size. see
+	// MinimumSnapSize for more details
```

Comment:
(very very nitpick) for multi-line or multi-sentence comments, I think we prefer to use usual capitalization rules. It's just for quick un-punctuated notes that we don't capitalize, at least from what I've seen. For (silly) example:

```go
// The following line sets foo to be equal to bar. This is
// important to ensure correctness.
foo = bar // make sure foo is correct
// foo should be correct now
```